tools/merger/merge.ts

```diff
@@ -0,0 +1,9 @@
+import OpenApiMerger from "./OpenApiMerger";
+
+
+async function main() {
+    const merger = new OpenApiMerger('../spec/OpenSearch.openapi.yaml');
+    merger.merge('../builds/OpenSearch.latest.yaml');
+}
```

Comment:
Tools should take source and destination as parameters rather than hard-code for this repo.

Reply:
Resolved by https://github.com/opensearch-project/opensearch-api-specification/pull/208/commits/0958319e2eca591c3d3f356413b045f78b2785c0